group_vars/orangelight/alma_qa.yml

```diff
@@ -0,0 +1,120 @@
+---
+passenger_server_name: "catalog-alma-qa.*"
+passenger_app_root: "/opt/orangelight/current/public"
+passenger_app_env: "alma-qa"
+passenger_extra_config: '{{ lookup("file", "roles/pulibrary.orangelight/templates/nginx_extra_config")  }}'
+rails_app_name: "orangelight"
+rails_app_directory: "orangelight"
+rails_app_symlinks: []
+rails_app_dependencies:
+  - libpq-dev
+  - pkg-config
+  - libtool
+  - autoconf
+passenger_ruby: "/usr/bin/ruby2.6"
+ruby_version_override: "ruby2.6"
+bundler_version: "2.1.4"
+postgresql_is_local: false
+postgres_host: "lib-postgres3.princeton.edu"
+postgres_version: 10
+ol_db_host: "{{ postgres_host }}"
+ol_db_name: "{{ vault_ol_alma_qa_db_name }}"
+ol_db_user: "{{ vault_ol_alma_qa_db_user}}"
+ol_db_password: "{{ vault_ol_alma_qa_db_password }}"
+ol_secret_key: "{{ vault_ol_alma_qa_secret_key }}"
+ol_graph_api_url: "{{ vault_ol_alma_qa_graph_api_url }}"
+ol_figgy_url: "{{ vault_ol_staging_figgy_url }}"
+ol_feedback_to: "{{ vault_ol_feedback_to }}"
+ol_feedback_cc: "{{ vault_ol_feedback_cc }}"
+ol_newrelic_key: ""
+ol_newrelic_app_name: ""
+ol_honeybadger_key: ""
+ol_rabbit_user: "{{ vault_figgy_alma_qa_rabbit_user }}"
+ol_rabbit_password: "{{ vault_figgy_alma_qa_rabbit_password }}"
+ol_rabbit_host: "figgy-staging1.princeton.edu"
+ol_rabbit_server: "amqp://{{ ol_rabbit_user }}:{{ ol_rabbit_password }}@{{ ol_rabbit_host }}:5672"
+voyager_ub_id: "{{ vault_voyager_ub_id }}"
```

Comment:
for the qa-server we will have alma variables. I think we need to remove the voyager ones.

Reply:
The alma keys have been added in the lastpass